### DISCUSSION TITLE
Additional Themes or Option for Custom Themes

### DISCUSSION THREAD

**Discussion by RazaHussainAziz** [cpeditor/cpeditor#1119] [Ideas]
Can you add Dark+ theme in CP Editor ? Like VsCode have a Darks+ theme.

**Comment by ouuan** [comment]
We planned to use KSyntaxHighlighting which provides [20+ themes](https://github.com/KDE/syntax-highlighting/tree/master/data/themes), but it's getting stuck (#516 and #1101), as there's not much progress on CP Editor recently.

**Comment by funcdfs** [comment]
check this: 
https://github.com/rogeryoungh/cpeditor/releases/tag/7.0.6

there is more theme, But this release has a lot of bugs, the theme is not easy to use, and the color matching is not complete （function name equal plain text）
